src/libopenrave/kinbody.cpp

```diff
@@ -3467,84 +3539,117 @@ void KinBody::_ComputeInternalInformation()
 
     try {
         // initialize all the mimic equations
-        for(int ijoints = 0; ijoints < 2; ++ijoints) {
-            vector<JointPtr>& vjoints = ijoints ? _vPassiveJoints : _vecjoints;
-            FOREACH(itjoint,vjoints) {
-                for(int i = 0; i < (*itjoint)->GetDOF(); ++i) {
-                    if( !!(*itjoint)->_vmimic[i] ) {
-                        std::string poseq = (*itjoint)->_vmimic[i]->_equations[0], veleq = (*itjoint)->_vmimic[i]->_equations[1], acceleq = (*itjoint)->_vmimic[i]->_equations[2]; // have to copy since memory can become invalidated
-                        (*itjoint)->SetMimicEquations(i,poseq,veleq,acceleq);
+        for(bool bPassiveJoints : {false, true}) {
```

Comment:
@tgn3000 i'm not sure all openrave users are using c++ with this fancy feature enabled...

Reply:
This is just c++11 for-loop. In `CMakeLists.txt` we already had
```
set(CMAKE_CXX_STANDARD 11)
```